### PR TITLE
teacher signup

### PR DESCRIPTION
**Pull request by emmayuan25**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -118,6 +121,8 @@ const App = () => {
                     <Route exact path='/about' component={ About } />
                     <Route exact path='/contact' component={ Contact } />
 
+                    <Route exact path='/teachersignup' component={ TeacherSignUp } />
```
Comment (crlimacastro) [inline src/App.js:124]: might want to hyphenate URL
e.g. /teacher-signup

File: src/pages/TeacherSignUp.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+        if (!checked) {
+            alert("must agree to terms/services and Data Use Policy to proceed.")
+        }
+        
+        if(schoolId && schoolName && schoolWebsite && checked){
+            // pass data to backend
+            const teacherData = { schoolId, schoolName, schoolWebsite, bio };
+            axios.post(`${REACT_APP_DATABASE_URL}/teachers/create`, teacherData)
+            .then(response => {
```
Comment (BrownMY) [inline src/pages/TeacherSignUp.js:53]: Lets make this an async

File: src/styles/teacher.signup.styles.css [hunk 3]
```diff
@@ -0,0 +1,221 @@
+/* Font */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,600;1,600&display=swap');
```
Comment (BrownMY) [inline src/styles/teacher.signup.styles.css:2]: this should be handled in the index, i dont think we'll need this here

**Review by crlimacastro — APPROVED** [review]

**Review by ldb297 — APPROVED** [review]
great work

**Pull request merged**